<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>医疗信息监控平台</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    /* 主窗口样式 */
    QWidget {
        background-color: #f0f5ff;
        font-family: 'Microsoft YaHei';
    }

    /* 面板样式 */
    QGroupBox {
        background-color: white;
        border: 1px solid #d1e3f8;
        border-radius: 8px;
        padding: 15px;
        margin-top: 10px;
    }

    QGroupBox::title {
        subcontrol-origin: margin;
        left: 10px;
        padding: 0 3px;
        color: #2b5876;
        font-weight: bold;
        font-size: 14px;
    }

    /* 标签样式 */
    QLabel {
        color: #2b5876;
        font-size: 14px;
        font-weight: bold;
        margin-bottom: 5px;
    }

    /* 文本编辑框样式 */
    QTextEdit {
        background-color: white;
        border: 1px solid #d1e3f8;
        border-radius: 6px;
        padding: 10px;
        font-size: 13px;
        color: #333333;
        selection-background-color: #4a90e2;
        selection-color: white;
    }

    /* 状态栏样式 */
    QLabel#statusBar {
        background-color: #e6f0ff;
        border-top: 1px solid #d1e3f8;
        padding: 8px;
        font-size: 12px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">
       font-size: 18px;
       font-weight: bold;
       color: #2b5876;
       qproperty-alignment: AlignCenter;
       margin-bottom: 15px;
      </string>
     </property>
     <property name="text">
      <string>医疗信息监控平台</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <item>
      <widget class="QGroupBox" name="leftGroupBox">
       <property name="title">
        <string>系统信息</string>
       </property>
       <layout class="QVBoxLayout" name="leftLayout">
        <item>
         <widget class="QLabel" name="networkInfoLabel">
          <property name="text">
           <string>网络连接信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>180</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="allUserLabel">
          <property name="text">
           <string>所有用户信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="allUserTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="rightGroupBox">
       <property name="title">
        <string>医疗数据</string>
       </property>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="allDoctorLabel">
          <property name="text">
           <string>所有医生信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="allDoctorTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="allPatientLabel">
          <property name="text">
           <string>所有患者信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="allPatientTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="caseGroupBox">
       <property name="title">
        <string>病例信息</string>
       </property>
       <layout class="QVBoxLayout" name="caseLayout">
        <item>
         <widget class="QLabel" name="allCaseLabel">
          <property name="text">
           <string>所有病例信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="allCaseTextEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>500</height>
           </size>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusBar">
     <property name="text">
      <string>服务器运行中 | IP: 127.0.0.1 | 端口: 8888 | 在线用户: 0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
